schema.sql

```diff
@@ -8,4 +6,6 @@ vet_clinic=# CREATE TABLE animals (
   neutered BOOLEAN,
   weight_kg DECIMAL(5,2)
 );
+vet_clinic=# ALTER TABLE animals ADD COLUMN species VARCHAR(255);
+ALTER TABLE
```

Comment:
- [ ] Also, the previous reviewer mentioned that we should see a code that adds a column to the `animals` table. I see that you attempted to add the code on lines 9 and 10 but these will not work because you have commented it out and it is of no effect. Please remove the `vet_clinic=#` text and let `ALTER TABLE animals ADD COLUMN species VARCHAR(255);` start line 9. Then remove line 10 completely. 

Essentially, adding this species column needs only the code below to do that. 
```sql
ALTER TABLE animals ADD species varchar(255);
```